src/Matches/Matches.js

```diff
@@ -91,29 +107,83 @@ class Matches extends React.Component {
+                        messageSenderName = this.state.currentName;
+                    }
+                    else {
+                        messageSenderName = this.state.messagedUserNames[i]
+                    }
+                    messagedUserItems.push(
+                        <div>
+                            <Link to={{pathname: '/viewfriendprofile', state: {id: this.state.userId, friendId: this.state.messagedUserIds[i]}}}>{this.state.messagedUserNames[i]}</Link>
+                            <button className='pos-user' 
+                                    key={pos_user}
+                                    value = {this.state.messagedUserIds[i]+"|"+this.state.messagedUserNames[i]} 
+                                    onClick = {this.selectUser}>
 
+                                {messageSenderName}: {this.state.messageContent[i]}
+                                timestamp: {this.state.timeStamp[i]}
+                            </button>  
+                        </div>
+                    )
+                }
+            }
+            if (this.state.notMessagedUserIds && this.state.notMessagedUserIds.length > 0){
+                for (var i = 0; i < this.state.notMessagedUserIds.length; i++){
+                    var pos_user = this.state.notMessagedUserIds[i];
+                    notMessagedUserItems.push(
+                        <div>
+                            <Link to={{pathname: '/viewfriendprofile', state: {id: this.state.userId, friendId: this.state.notMessagedUserIds[i]}}}>{this.state.notMessagedUserNames[i]}</Link>
+                            <button className='pos-user' 
+                                    key={pos_user} 
+                                    value = {this.state.notMessagedUserIds[i]+"|"+this.state.notMessagedUserNames[i]} 
+                                    onClick = {this.selectUser}>
+                                {this.state.notMessagedUserNames[i]}
+                            </button>  
+                        </div>
+                    )
+                }
+            }
+            if (this.state.messagedUserIds != [] && this.state.notMessagedUserIds.length != []){
+                matchingSection =  <h3 id='Matches-congrats'>
+                        <p>Congratulations, </p>
+                        <p>you have a match!</p> 
+                        <p>Messaged Users</p>
+                        <p>    
+                            {messagedUserItems}
+                        </p>
+                        <p>Not Messaged Users</p>
+                        <p> 
+                            {notMessagedUserItems}
+                        </p>
+                        </h3>
+            }
+            else if (messagedUserItems.length > 0) {
+                matchingSection =  <h3 id='Matches-congrats'>
+                        <p>You don't have any new matches</p>
+                        <p>Messaged Users</p>
```

Comment:
Maybe add some lines saying that you can continue chatting with your previous matches or wait until more users to join the community and check back later?